rnd/autogpt_server/autogpt_server/blocks/agent.py

```diff
@@ -79,10 +82,13 @@ class Input(BlockSchema):
         task: str
         input: str
         openai_api_key: BlockFieldSecret = BlockFieldSecret(key="openai_api_key")
-        enabled_components: list[str] = Field(default_factory=lambda: [OutputComponent.__name__])
+        enabled_components: list[str] = Field(
+            default_factory=lambda: [OutputComponent.__name__],
+            description="What components to enable [docs](https://docs.agpt.co/forge/components/built-in-components/)",
```

Comment:
"What components to enable docs"

what do you mean?

Reply:
I changed descriptions on multiple fields